samples/CommunityToolkit.Maui.Markup.Sample/HotReloadHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace CommunityToolkit.Maui.Markup.Sample;
+
+class HotReloadHandler : ICommunityToolkitHotReloadHandler
+{
+	public async void OnHotReload(IReadOnlyList<Type> types)
+	{
+		if (Application.Current?.MainPage is not Page mainPage)
```

Comment:
Main Page is not always the entry point. Rather use the Application.Windows as there may be multiple windows or the user has overridden the Application.CreateWindow - the better option anyways.

MainPage is mostly for compatibility and easy migration but breaks down if you do desktop things. MainPage also appears in the Windows. Typically Application.Current.Windows[0].Page is the MainPage, but if there are multiple windows then it could be any one.

Reply:
Good to know! I gotta shake off that old Xamarin.Forms mind-set 😁

I've update this example to loop through the `IReadOnlyList<Window?>`.